### DISCUSSION TITLE
How to unbind the back command from <Back>, H

### DISCUSSION THREAD

**Discussion by Momijie** [qutebrowser/qutebrowser#7289] [Q&A] [answered]
I'm having difficulties trying to unbind  the back command from <Back>,  H
Not exactly sure how I'm supposed to format it so I can unbind it.

Also is there a way to see how the defaults are binded so I can go about unbinding them?

**Comment by The-Compiler** [comment, marked as answer]
`:unbind H`. Using `:bind` without arguments shows you the current bindings.
